src/components/Map/MapComponent.tsx

```diff
@@ -0,0 +1,22 @@
+import 'leaflet/dist/leaflet.css';
+import React from 'react';
+import { MapContainer, TileLayer } from 'react-leaflet';
+
+import CircleComponent from './CircleComponent';
+import './MapComponent.scss';
+
+function MapComponent() {
+  return (
+    <MapContainer center={[50.05, 19.93]} zoom={12.5} className='map-container'>
```

Comment:
```suggestion
    <MapContainer center={[50.05, 19.93]} zoom={12.5} className='w-screen h-screen'>
```
Use styles directly in the `className` property using `tailwind`.